CHANGELOG.md

```diff
@@ -1,5 +1,282 @@
 # Changelog
 
+## 0.68-stable
+
+### Breaking
+
+
```

Comment:
Please note that this commit https://github.com/facebook/react-native/commit/0912ee179c210fb6b2ed9afbb3f2fbc5fb8a2bb3 was marked as internal but instead it's a breaking change. 

We should potentially mention that the `fallbackResource` parameter should be removed in order for the code to compile.

Reply:
Updated below